locations/spiders/hermes.py

```diff
@@ -0,0 +1,19 @@
+import scrapy
+
+from locations.dict_parser import DictParser
+
+
+class HermesSpider(scrapy.Spider):
+    name = "hermes"
+    item_attributes = {
+        "brand": "Hermes",
+        "brand_wikidata": "Q843887",
+    }
+    allowed_domains = ["hermes.com"]
+    start_urls = ["https://bck.hermes.com/stores?lang=en"]
+
+    def parse(self, response):
+        for store in response.json()["shops"]:
+            item = DictParser.parse(store)
```

Comment:
DictParser isn't able to fetch the coords from `geoCoordinates`, we can move it to `location` though and it should get them fine.

```suggestion
            store["store"]["location"] = store.pop("geoCoordinates")
            item = DictParser.parse(store)
```